src/compile/scale/range.ts

```diff
@@ -116,10 +116,13 @@ function getRangeStep(model: UnitModel, channel: 'x' | 'y'): number | SignalRef
       const binSignal = model.getName(vgField(fieldDef, {suffix: 'bins'}));
 
       // TODO: extract this to be range step signal
-      const binCount = `(${binSignal}.stop - ${binSignal}.start) / ${binSignal}.step`;
       const sizeType = getSizeType(channel);
       const sizeSignal = model.getName(sizeType);
```

Comment:
Just wondering, why do we need this at all? What happens if we delete all of it and never generate a rangeStep for binned fields?

Reply:
We may need it for ordinal data so try that.

Reply:
Ahh, looks like we do need the rangeStep signal if you look at the updated examples.